receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecscontainermetrics
+
+import (
+	"time"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	"go.opentelemetry.io/collector/consumer/consumerdata"
+)
+
+// metricDataAccumulator defines the accumulator
+type metricDataAccumulator struct {
+	md []*consumerdata.MetricsData
+}
+
+// getMetricsData generates OT Metrics data from task metadata and docker stats
+func (acc *metricDataAccumulator) getMetricsData(containerStatsMap map[string]ContainerStats, metadata TaskMetadata) {
+
+	taskMetrics := ECSMetrics{}
+	timestamp := timestampProto(time.Now())
+	taskResources := taskResources(metadata)
```

Comment:
nit: `taskResource` would be more accurate. Same with `containerResources` (-> `containerResource`)

Reply:
Updated.